### PR TITLE
48: Apply Grumphp refactoring to the library

### PR DESCRIPTION
**Pull request by RidRack**

### CODE REVIEW

File: tests/CarrierTest.php [hunk 1]
```diff
@@ -107,159 +132,349 @@ public function testRoyalmailClassRealValuesAll()
...
-        foreach ($this->dataClass->getMappingMethodToMeta() as $array => $data) {
+        foreach ($this->_dataClass->getMappingMethodToMeta()
+            as $array => $data) {
             $methodNotExist = false;
-            foreach ($this->dataClass->getMappingCleanNameToMethod() as $method => $methodData) {
+            foreach ($this->_dataClass->getMappingCleanNameToMethod()
+                as $method => $methodData) {
                 // Check the the names are equal
-                if ($data[self::METHOD_NAME_ROW_META_CSV] == $methodData[self::METHOD_NAME_ROW_CLEANNAME_CSV]) {
+                if ($data[self::METHOD_NAME_ROW_META_CSV] == $methodData[self::METHOD_NAME_ROW_CLEANNAME_CSV]
```
Comment (RidRack) [inline tests/CarrierTest.php]: One of the four lines that could not be shrunk. Moving the equals line to the next line causes grumphp to throw the "Each line in a multi-line IF statement must begin with a boolean operator" error.

File: src/Data.php [hunk 2]
```diff
@@ -299,43 +339,46 @@ private function getMethodToPrice($package_weight, $sortedMethodToMeta, $mapping
...
+     * @param array $sortedZoneToMethods - Sorted array of zone to methods
+     * @param array $mappingMethodToMeta - Sorted array of methods to the meta
      *
      * @return array
      */
-    private function getMethodToMetaAll($sortedZoneToMethods, $mappingMethodToMeta)
+    private function _getMethodToMetaAll($sortedZoneToMethods, $mappingMethodToMeta)
     {
-        $mappingZoneMethodData = [];
+        $mappingZoneMethodData = array();
```
Comment (bobbyshaw) [inline src/Data.php]: We can use short array syntax now as we're not supporting <5.4 going forward.

File: tests/CarrierTest.php [hunk 3]
```diff
@@ -10,37 +34,52 @@ class CarrierTest extends \PHPUnit_Framework_TestCase
     const METHOD_CLEAN_NAME_ROW_CLEANNAME_CSV = 1;
     const METHOD_CLEAN_NAME_ROW_CLEANNAMEGROUP_CSV = 1;
     const METHOD_CLEAN_NAME_GROUP_CLEANNAME_CSV = 0;
-    const METHOD_CLEAN_NAME_GROUP_CLEANNAMEGROUP_CSV = 0;
+    const METHOD_CLEAN_NAME_GROUP_CLNNAMEGRP_CSV = 0;
```
Comment (bobbyshaw) [inline tests/CarrierTest.php]: Hmm

File: tests/CarrierTest.php (hunk 1 above)
Comment (bobbyshaw) [inline tests/CarrierTest.php]: Did you find the config change to bump it up?

**Comment by bobbyshaw** [comment]
👍

File: tests/CarrierTest.php (hunk 1 above)
Comment (RidRack) [inline tests/CarrierTest.php]: No, I didn't. Might have to edit the psr-2 requirement, but I couldn't find somewhere to change it from grumphp directly.

File: tests/CarrierTest.php (hunk 3 above)
Comment (RidRack) [inline tests/CarrierTest.php]: 😢

**Pull request merged**